<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>432</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../4月21日/img/背景图.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="ton_1"/>
   <addaction name="separator"/>
   <addaction name="ton_2"/>
   <addaction name="separator"/>
   <addaction name="ton_3"/>
   <addaction name="separator"/>
   <addaction name="ton_4"/>
   <addaction name="separator"/>
   <addaction name="ton_5"/>
  </widget>
  <action name="ton_1">
   <property name="icon">
    <iconset>
     <normaloff>../../../4月21日/img/图标-1.png</normaloff>../../../4月21日/img/图标-1.png</iconset>
   </property>
   <property name="text">
    <string>各区二手房价均价分析</string>
   </property>
   <property name="toolTip">
    <string>各区二手房价均价分析</string>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </action>
  <action name="ton_2">
   <property name="icon">
    <iconset>
     <normaloff>../../../4月21日/img/图标-2.png</normaloff>../../../4月21日/img/图标-2.png</iconset>
   </property>
   <property name="text">
    <string>各区二手房数量所占比例</string>
   </property>
   <property name="toolTip">
    <string>各区二手房数量所占比例</string>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </action>
  <action name="ton_3">
   <property name="icon">
    <iconset>
     <normaloff>../../../4月21日/img/图标-3.png</normaloff>../../../4月21日/img/图标-3.png</iconset>
   </property>
   <property name="text">
    <string>全市二手房装修程度分析</string>
   </property>
   <property name="toolTip">
    <string>全市二手房装修程度分析</string>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </action>
  <action name="ton_4">
   <property name="icon">
    <iconset>
     <normaloff>../../../4月21日/img/图标-4.png</normaloff>../../../4月21日/img/图标-4.png</iconset>
   </property>
   <property name="text">
    <string>热门户型均价分析</string>
   </property>
   <property name="toolTip">
    <string>热门户型均价分析</string>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </action>
  <action name="ton_5">
   <property name="icon">
    <iconset>
     <normaloff>../../../4月21日/img/图标-5.png</normaloff>../../../4月21日/img/图标-5.png</iconset>
   </property>
   <property name="text">
    <string>二手房售价预测</string>
   </property>
   <property name="toolTip">
    <string>二手房售价预测</string>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
